README.md

```diff
@@ -1,5 +1,52 @@
-# curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+# My Portofolio
```

Comment:
**[Optional]**

Good job @aimemalaika for adding your Project's name. Kindly fix the spellings by replacing `Portofolio` with `Portfolio`